topkapi_test.go

```diff
@@ -358,3 +360,50 @@ func split(words []string, splits int) [][]string {
 
 	return slices
 }
+
+func gcd(a, b int64) int64 {
+	for b > 0 {
+		a, b = b, a%b
+	}
+	return a
+}
+
+func semiRandomNumber(i int64) int64 {
```

Comment:
It's not clear why you need a custom RNG and can't use the default in go?

Reply:
thats for testing i took it from eventdb code

Reply:
but why not the standard `rand.Int63n(i)`?

Reply:
don't know i did not bother looking at what the test did. Just wanted to test against what we have in eventdb so i can compare